### ISSUE TITLE
[Bug] g.gui.timeline plot fails in GRASS 8.3.2

### ISSUE THREAD

**Issue by mrtnlowry** [OSGeo/grass#4680]
## Describe the bug
Having managed to build an STRDS I want to check the relationships of the rasters but each time I try g.gui.timeline crashes with an error whilst obtaining timezone information.

## To reproduce

1. Start the gui from main menu
2. Select any STRDS dataset
3. Click 'Draw'

## System description
- Operating System: Windows 11
- GRASS GIS version: 8.3.2
- Python: 3.9.18                                                                  
- wxPython: 4.2.1                                                                 

Traceback:
[grass_cmd_output.txt](https://github.com/user-attachments/files/17691223/grass_cmd_output.txt)

```python
Traceback (most recent call last):
  File "C:\PROGRA~1\QGIS33~1.5\apps\grass\grass83\gui\wxpyth
on\tplot\frame.py", line 1120, in OnRedraw

self._redraw()
  File "C:\PROGRA~1\QGIS33~1.5\apps\grass\grass83\gui\wxpyth
on\tplot\frame.py", line 1136, in _redraw

self._drawFigure()
  File "C:\PROGRA~1\QGIS33~1.5\apps\grass\grass83\gui\wxpyth
on\tplot\frame.py", line 707, in _drawFigure

self.fig.autofmt_xdate()
  File "C:\PROGRA~1\QGIS33~1.5\apps\Python39\lib\site-
packages\matplotlib\figure.py", line 274, in autofmt_xdate

for label in self.axes[0].get_xticklabels(which=which):
  File "C:\PROGRA~1\QGIS33~1.5\apps\Python39\lib\site-
packages\matplotlib\axes\_base.py", line 75, in wrapper

return get_method(self)(*args, **kwargs)
  File "C:\PROGRA~1\QGIS33~1.5\apps\Python39\lib\site-
packages\matplotlib\axis.py", line 1242, in get_ticklabels

return self.get_majorticklabels()
  File "C:\PROGRA~1\QGIS33~1.5\apps\Python39\lib\site-
packages\matplotlib\axis.py", line 1201, in
get_majorticklabels

ticks = self.get_major_ticks()
  File "C:\PROGRA~1\QGIS33~1.5\apps\Python39\lib\site-
packages\matplotlib\axis.py", line 1371, in get_major_ticks

numticks = len(self.get_majorticklocs())
  File "C:\PROGRA~1\QGIS33~1.5\apps\Python39\lib\site-
packages\matplotlib\axis.py", line 1277, in
get_majorticklocs

return self.major.locator()
  File "C:\PROGRA~1\QGIS33~1.5\apps\Python39\lib\site-
packages\matplotlib\dates.py", line 1340, in __call__

dmin, dmax = self.viewlim_to_dt()
  File "C:\PROGRA~1\QGIS33~1.5\apps\Python39\lib\site-
packages\matplotlib\dates.py", line 1125, in viewlim_to_dt

return num2date(vmin, self.tz), num2date(vmax, self.tz)
  File "C:\PROGRA~1\QGIS33~1.5\apps\Python39\lib\site-
packages\matplotlib\dates.py", line 528, in num2date

return _from_ordinalf_np_vectorized(x, tz).tolist()
  File "C:\PROGRA~1\QGIS33~1.5\apps\Python39\lib\site-
packages\numpy\lib\function_base.py", line 2329, in __call__

return self._vectorize_call(func=func, args=vargs)
  File "C:\PROGRA~1\QGIS33~1.5\apps\Python39\lib\site-
packages\numpy\lib\function_base.py", line 2412, in
_vectorize_call

outputs = ufunc(*inputs)
  File "C:\PROGRA~1\QGIS33~1.5\apps\Python39\lib\site-
packages\matplotlib\dates.py", line 357, in _from_ordinalf

dt = dt.replace(tzinfo=dateutil.tz.gettz('UTC'))
  File "C:\PROGRA~1\QGIS33~1.5\apps\Python39\lib\site-
packages\dateutil\tz\tz.py", line 1557, in __call__

rv = self.nocache(name=name)
  File "C:\PROGRA~1\QGIS33~1.5\apps\Python39\lib\site-
packages\dateutil\tz\tz.py", line 1648, in nocache

tz = tzwin(name)
  File "C:\PROGRA~1\QGIS33~1.5\apps\Python39\lib\site-
packages\dateutil\tz\win.py", line 221, in __init__

keydict = valuestodict(tzkey)
  File "C:\PROGRA~1\QGIS33~1.5\apps\Python39\lib\site-
packages\dateutil\tz\win.py", line 364, in valuestodict

value = tz_res.name_from_string(value)
  File "C:\PROGRA~1\QGIS33~1.5\apps\Python39\lib\site-
packages\dateutil\tz\win.py", line 122, in name_from_string

return self.load_name(offset)
  File "C:\PROGRA~1\QGIS33~1.5\apps\Python39\lib\site-
packages\dateutil\tz\win.py", line 91, in load_name

nchar = self.LoadStringW(self._tzres._handle, offset,
lpBuffer, 0)
ctypes
.
ArgumentError
:
argument 1: <class 'OverflowError'>: int too long to convert
```

**Comment by neteler** [comment]
Please edit and copy/paste the error message with further details like your operating system etc.

**Comment by mrtnlowry** [comment]
Sorry, forgot to remove comment tags. All should now be visible.

**Comment by petrasovaa** [comment]
This is a known bug (unfortunately without a known solution...). It could be some packaging issue. See a workaround: https://github.com/OSGeo/grass/issues/2109#issuecomment-1264651550

**Comment by petrasovaa** [comment]
Closed as duplicate of #2109.

**Comment by mrtnlowry** [comment]
Okay, thanks for the workaround - it works.
